.github/CONTRIBUTING.md

```diff
@@ -1,2 +1,63 @@
-#  Contributing to WRF-Hydro
-Please note that we are currently in the process of developing contributing guidelines and testing procedures for the public WRF-Hydro repository.  These will be available at the time of the public announcement of the version 5 release of the code.  While we appreciate feedback submitted via issues on GitHub we ask that people do not submit pull requests at this time.  
+# Contributing to WRF-Hydro
+All bug reports, bug fixes, code, and documentation contributions are welcome and encouraged. However, guidelines need to be established and followed to ensure that community contributions can be effectively incorporated.
+
+Please read the following guidelines carefully before contributing. 
+
+# Guidelines
+
+**Failure to adhere to the following guidelines may result in your contributions being permanently rejected if they are unable to be merged.**
```

Comment:
This is related to the 2 repo system. If someone does considerable development work on a piece of code that is being completely refactored by the OWP devs, then indeed their contribution may be permanently rejected because it does not fit the new data structure/paradigm

Reply:
Makes good sense. "Failure to communicate with the core development team and/or adhere to the following guidelines..." ?